kylin-spark-project/kylin-spark-engine/src/main/java/org/apache/kylin/engine/spark/source/CsvSource.java

```diff
@@ -59,7 +59,12 @@ public CsvSource(KylinConfig config) {
                         && (parameters != null && parameters.get("separator") == null)) {
                     path = "file:///" + new File(getUtMetaDir(),
                             "../../examples/test_case_data/parquet_test/data/" + table.identity() + ".csv")
-                                    .getAbsolutePath();
+                            .getAbsolutePath();
+                    separator = "";
+                } else if (kylinConfig.getDeployEnv().equals("LOCAL")) {
+                    path = "file:///" + new File(getUtMetaDir(),
+                            "../parquet_test/data/" + table.identity() + ".csv")
```

Comment:
why ut path is "../../examples/test_case_data/parquet_test/data/", but local path is  "../parquet_test/data/"?
can they use same path?

Reply:
Agreed, these two path confusing me too.